src/main/java/com/example/nasastealphoto/model/Camera.java

```diff
@@ -0,0 +1,51 @@
+package com.example.nasastealphoto.model;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.Cascade;
+import org.hibernate.annotations.CascadeType;
+import org.hibernate.annotations.CreationTimestamp;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "cameras")
+public class Camera {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(unique = true)
+    private Long nasaId;
+
+    @CreationTimestamp
+    private LocalDateTime createdAt;
+
+    @ToString.Exclude
+    @JsonBackReference
+    @Setter(value = AccessLevel.PRIVATE)
+    @OneToMany(mappedBy = "camera")
+    @Cascade(CascadeType.PERSIST)
```

Comment:
You can specified cascade type in **@OneToMany** annotation to make it more clear

Reply:
Fixed